liubov.tarasenko/homework7/image.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'pry'
+require 'faraday'
+
+class Image
+  def download(url)
+    @url = url
+    ext = @url.split('.').last
+    response = Faraday.get @url
+    raise ArgumentError, "code #{response.status}" if response.status != 200
+    raise TypeError, 'Not an image' unless type_correct?(response.headers['content-type'])
+
+    img = response.body
+    save(img, ext)
```

Comment:
why don't you actually use `response.headers['content-type']` as ext here?

Reply:
because 'content-type' usually is something like 'image/png' and theoretically can be different formats

Reply:
fair enough. lemme check with @Flippylolz  then